python/cudf/cudf/core/series.py

```diff
@@ -2328,22 +2328,22 @@ def __invert__(self):
     @copy_docstring(CategoricalAccessor.__init__)  # type: ignore
     @property
     def cat(self):
-        return CategoricalAccessor(column=self._column, parent=self)
+        return CategoricalAccessor(parent=self)
```

Comment:
Should we just name it `CategoricalMethods` at this point?

Reply:
For Pandas compat, we need to keep the name `CategoricalAccessor` unfortunately